.circleci/config.yml

```diff
@@ -192,6 +192,18 @@ jobs:
       - *makesourcetarball
       - *builddebianpackage
 
+  build-securedrop-proxy-buster:
+    docker:
+      - image: circleci/python3.7-buster
```

Comment:
ahh `circleci/python3.7-buster` -> `circleci/python:3.7-buster`

Reply:
(that's why CI was failing, this was confusing me)

Reply:
thanks! :star2: for you